### DISCUSSION TITLE
"webpack": "^5.74.0"   document.body is null

### DISCUSSION THREAD

**Discussion by HSQCoollaughing** [webpack/webpack#16397] [Q&A]
# Bug report

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  
Node.js version: 
Operating System: 
Additional tools:

**Comment by alexander-akait** [comment]
Please fill out the issue template
